classes/support/adapters/BlogPostXmlAdapter.php

```diff
@@ -53,6 +53,12 @@ public function getDataValue($item, $dataItem)
                 $item->appendChild($creator);
             break;
 
+            case 'media:thumbnail':
+                $item->setAttribute('url', \Config::get('app.url').ltrim($dataItem->featured_images[0]->path,'/'));
```

Comment:
Please add spaces between the method parameters and also when using the dot for string concatenation. It is hard to read because it is very similar to the C++ class method call.